app/uk/gov/hmrc/agentclientrelationshipsfrontend/views/journey/clientFactPartials/Email.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.config.AppConfig
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.journey.routes
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.models.common.KnownFactsConfiguration
+@import uk.gov.hmrc.agentclientrelationshipsfrontend.models.journey.AgentJourneyRequest
+@import uk.gov.hmrc.govukfrontend.views.Implicits.RichSelect
+
+@this(
+        formWithCSRF: FormWithCSRF,
+        govukInput: GovukInput,
+        govukButton: GovukButton
+)
+
+@(form: Form[?], field: KnownFactsConfiguration, messageKeyPrefix: String)(implicit request: AgentJourneyRequest[?], messages: Messages)
+
+@key = @{messageKeyPrefix}
+@validOptions = @{field.validOptions.getOrElse(Seq.empty)}
+@formWithCSRF(action = routes.EnterClientFactController.onSubmit(request.journey.journeyType)) {
+
+    @govukInput(Input(
+        id = field.name,
+        name = field.name,
+        hint = if messages.isDefinedAt(s"$messageKeyPrefix.hint") then Some(Hint(content =
+            Text(messages(s"$messageKeyPrefix.hint")))) else None,
+        label = Label(content = Text(messages(s"$key.label")), isPageHeading = true, classes = "govuk-label--xl"),
+        classes = "govuk-!-margin-bottom-4"
+    )
```

Comment:
```suggestion
    ).withFormField(form(field.name))
```
if you add `.withFormField(form(field.name))` then the template will render the inline error message as well as the error summary link